spaceship/lib/spaceship/tunes/app_version.rb

```diff
@@ -557,7 +557,10 @@ def setup_screenshots
 
       # generates the nested data structure to represent screenshots
       def setup_screenshots_for(row)
-        display_families = row.fetch("displayFamilies", {}).fetch("value", nil)
+        display_families_hash = row.fetch("displayFamilies", {})
```

Comment:
If the first row.fetch would be `nil`, it should fallback to `{}` as the default value. I think `row` might be nil at this point, causing the crash.

Have you actually tried this fix locally? Could you do so, and if it doesn't work, revert the changes, and add a new line as the first line of the method
```ruby
def setup_screenshots_for(row)
  return if row.nil?
  display_famililes ...
```